packages/earth-admin/src/services/organizations.tsx

```diff
@@ -37,3 +37,20 @@ export const getOrganization = (organizationQuery: string) => {
     method: 'get',
   });
 };
+
+export const updateOrganization = async (organizationID: string, organization, group: string) => {
+  return await OrganizationAPIService.request({
+    url: `/organizations/${organizationID}?group=${group}`,
+    method: 'put',
+    data: organization,
+  });
+};
+
+export const handleOrganizationForm = async (
+  newOrg: boolean,
+  organization,
+  organizationID: string,
+  group: string
+) => {
+  newOrg ? null : await updateOrganization(organizationID, organization, group);
+};
```

Comment:
I have mixed feelings about this function's purpose. It seems like you want to reuse it in the future when we add "create org" functionality.
I think it creates unnecessary abstraction, why not use `updateOrganization(...)` directly and `createOrganizarion(...)` when it's available.